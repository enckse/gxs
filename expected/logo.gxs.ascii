
. . . . . . . . . . . . . . 
                            
. . . . . . . . . . . . . . 
     a a a a                
. . . . . . . . . . . . . . 
   a                        
. . . . . . . . . . . . . . 
   a     a a                
. . . . . . . . . . . . . . 
   a     b a     b          
. . . . . . . . . . . . . . 
     a a a b   b            
. . . . . . . . . . . . . . 
             b              
. . . . . . . . . . . . . . 
           b   b c c c c    
. . . . . . . . . . . . . . 
         b     c b          
. . . . . . . . . . . . . . 
                 c c c      
. . . . . . . . . . . . . . 
                       c    
. . . . . . . . . . . . . . 
               c c c c      
. . . . . . . . . . . . . . 

---
color: a => green (count: 14)
color: b => red (count: 9)
color: c => blue (count: 13)
