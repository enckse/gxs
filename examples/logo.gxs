palette => {
    g => green
    x => red
    s => blue
    - => NONE
}
mode => {xstitch}
pattern => {
    -gggg
    g
    g--gg
    g---g
    -gggg
}
action => {commit}
mode => {xstitch}
offset => {3x3}
pattern => {
    x---x
    -x-x
    --x
    -x-x
    x---x
}
action => {commit}
mode => {xstitch}
offset => {6x6}
pattern => {
    -ssss
    s
    -sss
    ----s
    ssss-
}
action => {commit}
