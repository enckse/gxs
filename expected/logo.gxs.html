<!doctype html>
<html lang="en">
<head>
<meta charset="UTF-8">
<style>
.container {
  background: white;
  display: inline-block;
  border: 1px dotted black;
}
.grid {
  display: grid;
  grid-template-columns: repeat(12, 10px);
  grid-template-rows: repeat(12, 10px);
  grid-gap: 1px;
}
.cell {
  justify-content: center;
  align-items: center;
  display: flex;
  font-family: Arial;
  font-size: 4pt;
  font-weight: bold;
  background: white;
}
.main {
  margin-left: 10px;
  padding: 0px 10px;
}
</style>
    </head>
    <body>
        <div class="main">
  <div class="container">
  <div class="grid" id="grid">
      <div class="cell" style="" id="0000x0000"></div>
      <div class="cell" style="" id="0001x0000">1</div>
      <div class="cell" style="" id="0002x0000">2</div>
      <div class="cell" style="" id="0003x0000">3</div>
      <div class="cell" style="" id="0004x0000">4</div>
      <div class="cell" style="" id="0005x0000">5</div>
      <div class="cell" style="" id="0006x0000">6</div>
      <div class="cell" style="" id="0007x0000">7</div>
      <div class="cell" style="" id="0008x0000">8</div>
      <div class="cell" style="" id="0009x0000">9</div>
      <div class="cell" style="" id="0010x0000">10</div>
      <div class="cell" style="" id="0011x0000">11</div>
      <div class="cell" style="" id="0000x0001">1</div>
      <div class="cell" style="" id="0001x0001"></div>
      <div class="cell" style="background-color:  rgb(5, 101, 23)" id="0002x0001"></div>
      <div class="cell" style="background-color:  rgb(5, 101, 23)" id="0003x0001"></div>
      <div class="cell" style="background-color:  rgb(5, 101, 23)" id="0004x0001"></div>
      <div class="cell" style="background-color:  rgb(5, 101, 23)" id="0005x0001"></div>
      <div class="cell" style="" id="0006x0001"></div>
      <div class="cell" style="" id="0007x0001"></div>
      <div class="cell" style="" id="0008x0001"></div>
      <div class="cell" style="" id="0009x0001"></div>
      <div class="cell" style="" id="0010x0001"></div>
      <div class="cell" style="" id="0011x0001"></div>
      <div class="cell" style="" id="0000x0002">2</div>
      <div class="cell" style="background-color:  rgb(5, 101, 23)" id="0001x0002"></div>
      <div class="cell" style="" id="0002x0002"></div>
      <div class="cell" style="" id="0003x0002"></div>
      <div class="cell" style="" id="0004x0002"></div>
      <div class="cell" style="" id="0005x0002"></div>
      <div class="cell" style="" id="0006x0002"></div>
      <div class="cell" style="" id="0007x0002"></div>
      <div class="cell" style="" id="0008x0002"></div>
      <div class="cell" style="" id="0009x0002"></div>
      <div class="cell" style="" id="0010x0002"></div>
      <div class="cell" style="" id="0011x0002"></div>
      <div class="cell" style="" id="0000x0003">3</div>
      <div class="cell" style="background-color:  rgb(5, 101, 23)" id="0001x0003"></div>
      <div class="cell" style="" id="0002x0003"></div>
      <div class="cell" style="" id="0003x0003"></div>
      <div class="cell" style="background-color:  rgb(5, 101, 23)" id="0004x0003"></div>
      <div class="cell" style="background-color:  rgb(5, 101, 23)" id="0005x0003"></div>
      <div class="cell" style="" id="0006x0003"></div>
      <div class="cell" style="" id="0007x0003"></div>
      <div class="cell" style="" id="0008x0003"></div>
      <div class="cell" style="" id="0009x0003"></div>
      <div class="cell" style="" id="0010x0003"></div>
      <div class="cell" style="" id="0011x0003"></div>
      <div class="cell" style="" id="0000x0004">4</div>
      <div class="cell" style="background-color:  rgb(5, 101, 23)" id="0001x0004"></div>
      <div class="cell" style="" id="0002x0004"></div>
      <div class="cell" style="" id="0003x0004"></div>
      <div class="cell" style="background-color:  rgb(199, 43, 59)" id="0004x0004"></div>
      <div class="cell" style="background-color:  rgb(5, 101, 23)" id="0005x0004"></div>
      <div class="cell" style="" id="0006x0004"></div>
      <div class="cell" style="" id="0007x0004"></div>
      <div class="cell" style="background-color:  rgb(199, 43, 59)" id="0008x0004"></div>
      <div class="cell" style="" id="0009x0004"></div>
      <div class="cell" style="" id="0010x0004"></div>
      <div class="cell" style="" id="0011x0004"></div>
      <div class="cell" style="" id="0000x0005">5</div>
      <div class="cell" style="" id="0001x0005"></div>
      <div class="cell" style="background-color:  rgb(5, 101, 23)" id="0002x0005"></div>
      <div class="cell" style="background-color:  rgb(5, 101, 23)" id="0003x0005"></div>
      <div class="cell" style="background-color:  rgb(5, 101, 23)" id="0004x0005"></div>
      <div class="cell" style="background-color:  rgb(5, 101, 23);background-color:  rgb(199, 43, 59)" id="0005x0005"></div>
      <div class="cell" style="" id="0006x0005"></div>
      <div class="cell" style="background-color:  rgb(199, 43, 59)" id="0007x0005"></div>
      <div class="cell" style="" id="0008x0005"></div>
      <div class="cell" style="" id="0009x0005"></div>
      <div class="cell" style="" id="0010x0005"></div>
      <div class="cell" style="" id="0011x0005"></div>
      <div class="cell" style="" id="0000x0006">6</div>
      <div class="cell" style="" id="0001x0006"></div>
      <div class="cell" style="" id="0002x0006"></div>
      <div class="cell" style="" id="0003x0006"></div>
      <div class="cell" style="" id="0004x0006"></div>
      <div class="cell" style="" id="0005x0006"></div>
      <div class="cell" style="background-color:  rgb(199, 43, 59)" id="0006x0006"></div>
      <div class="cell" style="" id="0007x0006"></div>
      <div class="cell" style="" id="0008x0006"></div>
      <div class="cell" style="" id="0009x0006"></div>
      <div class="cell" style="" id="0010x0006"></div>
      <div class="cell" style="" id="0011x0006"></div>
      <div class="cell" style="" id="0000x0007">7</div>
      <div class="cell" style="" id="0001x0007"></div>
      <div class="cell" style="" id="0002x0007"></div>
      <div class="cell" style="" id="0003x0007"></div>
      <div class="cell" style="" id="0004x0007"></div>
      <div class="cell" style="background-color:  rgb(199, 43, 59)" id="0005x0007"></div>
      <div class="cell" style="" id="0006x0007"></div>
      <div class="cell" style="background-color:  rgb(199, 43, 59)" id="0007x0007"></div>
      <div class="cell" style="background-color:  blue" id="0008x0007"></div>
      <div class="cell" style="background-color:  blue" id="0009x0007"></div>
      <div class="cell" style="background-color:  blue" id="0010x0007"></div>
      <div class="cell" style="background-color:  blue" id="0011x0007"></div>
      <div class="cell" style="" id="0000x0008">8</div>
      <div class="cell" style="" id="0001x0008"></div>
      <div class="cell" style="" id="0002x0008"></div>
      <div class="cell" style="" id="0003x0008"></div>
      <div class="cell" style="background-color:  rgb(199, 43, 59)" id="0004x0008"></div>
      <div class="cell" style="" id="0005x0008"></div>
      <div class="cell" style="" id="0006x0008"></div>
      <div class="cell" style="background-color:  blue" id="0007x0008"></div>
      <div class="cell" style="background-color:  rgb(199, 43, 59)" id="0008x0008"></div>
      <div class="cell" style="" id="0009x0008"></div>
      <div class="cell" style="" id="0010x0008"></div>
      <div class="cell" style="" id="0011x0008"></div>
      <div class="cell" style="" id="0000x0009">9</div>
      <div class="cell" style="" id="0001x0009"></div>
      <div class="cell" style="" id="0002x0009"></div>
      <div class="cell" style="" id="0003x0009"></div>
      <div class="cell" style="" id="0004x0009"></div>
      <div class="cell" style="" id="0005x0009"></div>
      <div class="cell" style="" id="0006x0009"></div>
      <div class="cell" style="" id="0007x0009"></div>
      <div class="cell" style="background-color:  blue" id="0008x0009"></div>
      <div class="cell" style="background-color:  blue" id="0009x0009"></div>
      <div class="cell" style="background-color:  blue" id="0010x0009"></div>
      <div class="cell" style="" id="0011x0009"></div>
      <div class="cell" style="" id="0000x0010">10</div>
      <div class="cell" style="" id="0001x0010"></div>
      <div class="cell" style="" id="0002x0010"></div>
      <div class="cell" style="" id="0003x0010"></div>
      <div class="cell" style="" id="0004x0010"></div>
      <div class="cell" style="" id="0005x0010"></div>
      <div class="cell" style="" id="0006x0010"></div>
      <div class="cell" style="" id="0007x0010"></div>
      <div class="cell" style="" id="0008x0010"></div>
      <div class="cell" style="" id="0009x0010"></div>
      <div class="cell" style="" id="0010x0010"></div>
      <div class="cell" style="background-color:  blue" id="0011x0010"></div>
      <div class="cell" style="" id="0000x0011">11</div>
      <div class="cell" style="" id="0001x0011"></div>
      <div class="cell" style="" id="0002x0011"></div>
      <div class="cell" style="" id="0003x0011"></div>
      <div class="cell" style="" id="0004x0011"></div>
      <div class="cell" style="" id="0005x0011"></div>
      <div class="cell" style="" id="0006x0011"></div>
      <div class="cell" style="background-color:  blue" id="0007x0011"></div>
      <div class="cell" style="background-color:  blue" id="0008x0011"></div>
      <div class="cell" style="background-color:  blue" id="0009x0011"></div>
      <div class="cell" style="background-color:  blue" id="0010x0011"></div>
      <div class="cell" style="" id="0011x0011"></div>
  </div>
</div>
        </div>
    </body>
</html>
